packages/patternfly-4/react-core/src/components/Accordion/Accordion.tsx

```diff
@@ -27,7 +27,11 @@ export const Accordion: React.FunctionComponent<AccordionProps> = ({
   const AccordionList: any = asDefinitionList ? 'dl' : 'div';
   return (
     <AccordionList className={css(styles.accordion, className)} aria-label={ariaLabel} {...props}>
-      <AccordionContext.Provider value={{ AccordionHeadingLevel: headingLevel, asDefinitionList }}>
+      <AccordionContext.Provider value={{
+        HeadingLevel: headingLevel,
```

Comment:
Do we need HeadingLevel other than in `ToggleContainer` because it looks like redundent

Reply:
Fixed.